### PR TITLE
EN-3403: remove assertion that kept polygons from being used. 

### PR DESCRIPTION
**Pull request by AEscobarCruz**
Now we can. Tested locally with both custom and modified shapefiles that have polygon only entries and custmn point datasets.

### CODE REVIEW

File: src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala [hunk 1]
```diff
@@ -108,15 +113,22 @@ class SpatialRegionCache(config: Config) extends MemoryManagingRegionCache[Spati
...
     geomColumnCache(resourceName) {
       logger.info(s"Populating geometry column name for resource $resourceName from soda fountain..")
-      val tryColumn = SodaResponse.check(sodaFountain.schema(resourceName), Status_OK).map { jSchema =>
-        val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject]
-                           .collect { case (k, v) if v.dyn.datatype.! == JString("multipolygon") => k }
-                           .toSeq
-        assert(geoColumns.length == 1, "There should only be one multipolygon column in region " + resourceName)
+      val tryColumn =
+        SodaResponse.check(sodaFountain.schema(resourceName), Status_OK).map { jSchema =>
+        val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject].collect {
```
Comment (urmilan) [inline src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala]: Nit: should be indented to one indent level in from `SodaResponse.check ...` (same indent level as the `assert` line below that's also in the same block)

Nit: end curly brace of `collect` should go on next line. ie.
```scala
val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject].collect { 
  case (k, v) if isAcceptedGeometry(v.dyn.datatype.!) => k
}.toSeq
```

File: src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala [hunk 2]
```diff
@@ -108,15 +113,22 @@ class SpatialRegionCache(config: Config) extends MemoryManagingRegionCache[Spati
...
+          case (k, v) if isAcceptedGeometry(v.dyn.datatype.!) => k}.toSeq
+
+
+          assert(geoColumns.length == 1, s"There should only be one column of " +
+            s"type $polygon or type $multiPolygon in region " + resourceName)
         geoColumns.head
```
Comment (urmilan) [inline src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala]: Nit: indent to same level as `assert`

File: src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala [hunk 3]
```diff
@@ -108,15 +113,22 @@ class SpatialRegionCache(config: Config) extends MemoryManagingRegionCache[Spati
...
-      }
+        }
```
Comment (urmilan) [inline src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala]: Nit: outdent to same level as corresponding opening brace (L117)

**Comment by urmilan** [comment]
Reviewed up to 8fea944

**Comment by urmilan** [comment]
:shipit: once we figure out why the checks failed.

**Comment by AEscobarCruz** [comment]
All checks pass but one test fails for a classpath not found reasons... Trying to figure it out.

**Comment by AEscobarCruz** [comment]
Merging, even if one tests is failing, but seems to be innocuous and the case for quite a while.

**Pull request merged**